src/mmw/js/src/data_catalog/controllers.js

```diff
@@ -14,10 +14,32 @@ var DataCatalogController = {
             'was_data_catalog_visible': true,
         });
 
-        var searchModel = new models.DataCatalogModel();
-        var resources = new models.ResourceCollection();
+        var form = new models.SearchForm();
 
-        var view = new views.DataCatalogWindow(searchModel, resources);
+        var catalogs = new models.Catalogs([
+            new models.Catalog({
```

Comment:
Don't need to specify `new models.Catalog` here, you can just give an array of objects and `new models.Catalogs([...])` will convert it to the right objects automatically.